phoenix-core/src/it/java/org/apache/phoenix/end2end/UpsertValuesIT.java

```diff
@@ -480,10 +488,14 @@ public void testBatchedUpsert() throws Exception {
             pstmt.addBatch();
             pstmt.executeBatch();
             conn.commit();
```

Comment:
```suggestion
            if (!autocommit) {
                conn.commit();
            }
```

Reply:
Isn't it harmless to call commit (a 2nd time) even `if (autocommit)`?

Reply:
I agree with your philosophy on this, I have done this as well.